DigitalLearningSolutions.Data.Migrations/202201111021_ChangeCompetencyAssessmentQuestionRoleRequirementsPrimaryKey.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202201111021)]
+    public class ChangeCompetencyAssessmentQuestionRoleRequirementsPrimaryKey: Migration
+    {
+        public override void Up()
+        {
+            Delete.PrimaryKey("PK_CompetencyAssessmentQuestionRoleRequirements")
+                .FromTable("CompetencyAssessmentQuestionRoleRequirements");
+            Delete.Column("ID").FromTable("CompetencyAssessmentQuestionRoleRequirements");
+            Create.PrimaryKey("PK_CompetencyAssessmentQuestionRoleRequirements")
+                .OnTable("CompetencyAssessmentQuestionRoleRequirements").Columns("SelfAssessmentID", "CompetencyID");
+        }
+
+        public override void Down()
+        {
+            Delete.PrimaryKey("PK_CompetencyAssessmentQuestionRoleRequirements")
+                .FromTable("CompetencyAssessmentQuestionRoleRequirements");
+            Alter.Table("CompetencyAssessmentQuestionRoleRequirements").AddColumn("ID").AsInt32().NotNullable()
+                .PrimaryKey().Identity();

```

Comment:
Given your comment on the next migration, should this Down() have `Create.PrimaryKey...`?

Reply:
Good point. When I made the other migration, I thought it would be good to leave this one as is for some reason. But now I think it would break anytime someone wanted to migrate down beyond this. I've updated it to include the correct primary key addition.